firebase_admin/messaging.py

```diff
@@ -754,6 +776,29 @@ def encode_aps(cls, aps):
                 result[key] = val
         return cls.remove_null_values(result)
 
+    @classmethod
+    def encode_aps_sound(cls, sound):
+        """Encodes an APNs sound configuration into JSON."""
+        if sound is None:
+            return None
+        if sound and isinstance(sound, six.string_types):
+            return sound
+        if not isinstance(sound, CriticalSound):
+            raise ValueError(
+                'Aps.sound must be a non-empty string or an instance of CriticalSound class.')
+        result = {
+            'name': _Validators.check_string('CriticalSound.name', sound.name, non_empty=True),
+            'volume': _Validators.check_number('CriticalSound.volume', sound.volume),
+        }
+        if sound.critical is True:
```

Comment:
Should we allow any truthy value and not just `True` specifically?

Reply:
In the past we've been pretty strict with our type checking -- something I plan to relax as we go on. Made the change as suggested.